test/iso/component-param.js

```diff
@@ -0,0 +1,179 @@
+import sinon from 'sinon';
+import {expect} from 'chai';
+import {
+  DefaultAndRequiredParam,
+  ExtraParamPassInChild,
+  ParamChild,
+  ParamParentApp,
+  NonPrimitiveTypeParamClass,
+  NonPrimitiveTypeParamString,
+  ShouldComponentUpdateParamsApp,
+} from '../fixtures/params-app';
+import {nextAnimationFrame, compactHtml} from '../utils';
+
+customElements.define(`param-child`, ParamChild);
+customElements.define(`param-parent-app`, ParamParentApp);
+customElements.define(`param-required-and-default-app`, DefaultAndRequiredParam);
+customElements.define(`extra-param-pass-in-child`, ExtraParamPassInChild);
+customElements.define(`non-primitive-type-param-class`, NonPrimitiveTypeParamClass);
+customElements.define(`non-primitive-type-param-string`, NonPrimitiveTypeParamString);
+customElements.define(`should-component-update-params-app`, ShouldComponentUpdateParamsApp);
+
+describe(`panel-params`, () => {
+  beforeEach(() => {
+    document.body = document.createElement(`body`);
+  });
+
+  it(`mounts a parent-child component`, async () => {
+    const el = new ParamParentApp();
+    el.connectedCallback();
+    document.body.appendChild(el);
+    await nextAnimationFrame();
+    await nextAnimationFrame();
+    expect(el.childNodes[0].innerHTML).to.equal(
+      compactHtml(`
+      <div>
+        <div id="str"></div>
+        <div id="num">0</div>
+        <div id="bool"></div>
+        <div id="obj">{}</div>
+        <div id="arr">[]</div>
+        <div id="map">{}</div>
+        <div id="set">{}</div>
+      </div>
+    `),
+    );
+  });
+
+  it(`updates parent will cause update in child`, async () => {
```

Comment:
more grammatical, something like
```suggestion
  it(`updates child whenever parent is updated`, async function() {
```